.circleci/config.yml

```diff
@@ -15,6 +15,10 @@ parameters:
     description: The name of the workflow to run
     type: string
     default: pipeline
+  e2e-base-url:
+    description: The base url for running end-to-end test
+    type: string
+    default: https://mui.com
```

Comment:
I'm concerned about this value. I'm afraid that if we inadvertently depend on this default, we would have each PR test on production, which wouldn't work.

```suggestion
```